GenesToPhenotypes.ipynb

```diff
@@ -425,11 +1570,173 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 51,
+   "execution_count": 22,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if os.path.isfile(csvfile.name):\n",
+    "    os.remove(csvfile.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to IDR "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Connected to IDR ...\n"
+     ]
+    }
+   ],
+   "source": [
+    "conn = connection('idr.openmicroscopy.org')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load the image "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Cep215_PCNT_NEDD1_20110505_Thu-1504_0_SIR_PRJ.dv \n"
+     ]
+    }
+   ],
+   "source": [
+    "image = conn.getObject(\"Image\", 1884831)\n",
```

Comment:
I would prefer here if the first image from the csv above was passed, instead of hard-coding this ID. If the csv is empty, then just show a warning and do not continue.

Not a blocker if too cumbersome to implement though. The ID can be copied from the output of the above cells, but it would be nicer if this is not necessary - it would also show the way to connect the two parts of the notebook.